index.html

```diff
@@ -0,0 +1,33 @@
+  <!DOCTYPE html>
+  <html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My portfolio</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css">
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  <main class="about-me">
+  <nav>
+  <header class="logo">
+  <h3>Welcome</h3>
+  <a href="" class="menu"><i class="fas fa-bars"></i></a>
+  </header>
+  </nav>
```

Comment:
### Kindly try to match the Figma design in the header:
- [ ] Kindly fix the top and button spacing to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/160151232-54765826-9fd9-46cd-882b-f44136a07860.png) | ![image](https://user-images.githubusercontent.com/24703171/160151504-9974e647-d9cc-4fa7-9e98-d6138efa8f7a.png) |